explainer-datamodel.md

```diff
@@ -1,236 +1,331 @@
-CSS properties that do not apply to text, that convert between typical text layout
-and other layouts, or that take normal flow content out of flow (e.g., `float`, `position`, 
-`display`, etc.). For the purposes of a formatted text object model, not all CSS properties
-can or should be supported. The guidelines for what CSS to support and what not to support 
-follow.
+The `format` function supports various CSS properties that influence how the text's lines
+will ultimately be positioned. There are also many CSS properties that do not apply to text,
+that convert between typical text layout and other layouts, or that take normal flow content 
+out of flow (e.g., `float`, `position`, `display`, etc.). For the purposes of a formatted 
+text object model, not all CSS properties can or should be supported. The guidelines for
+what CSS to support and what not to support follow.
 
-#### Focus on text-related CSS properties
+### Focus on text-related CSS properties
 
 We believe it makes sense to only support CSS properties that provide specific features
-for inline-level content (text) and the management of the text's container. Other properties,
-especially those that would change the layout characteristics of `FormattedTextRun`
-objects from their assumed inline-level nature, will not be supported. In this data model
-we want to keep the semantics of a `FormattedTextRun` consistent with the CSS that can be
-applied.
+for inline-level content (text) and the management of the text's container (the metadata
+parameter). Other properties, especially those that would change the layout characteristics 
+of text objects from their assumed inline-level nature, will not be supported. In the data
+model we keep the semantics of text runs consistent with the CSS that can be applied.
 
-Applying the above principle to `FormattedTextRun` objects means that `float` would not
+Applying the above principle means that `float` would not
 be supported because it has the effect of changing the object's computed display value 
-from inline to block (taking it out-of-flow). `position: absolute` also takes
-the object out of the normal flow. Likewise, the `width` and `height` properties would
+from inline to block (taking it out-of-flow). `position: absolute` as a metadata property also takes
+the object out of the normal flow. Likewise, the `width` and `height` properties on specific text runs would
 be ignored because they do not apply to inline-level elements. Attempting to set these 
-properties would have no effect. On the other hand, `padding` would be supported and 
-applied to `FormattedTextRun` objects in accordance with the rules of CSS inline layout,
+properties would have no effect. On the other hand, `padding` **would be supported** and 
+applied to text runs in accordance with the rules of CSS inline layout,
 e.g., the inline-direction `padding` values are taken into account while the 
 block-direction values are not.
```

Comment:
Not strictly part of your PR... but why not?   Elsewhere you mention wanting  to support a limited subset of `background` and `border`, so the block-direction padding is important for those cases.

Reply:
I think I am saying that: 'yes' padding is supported. The change is just to highlight that fact... Did I miss something?

Reply:
> inline-direction `padding` values are taken into account while the block-direction values are not

My point was that I think block-direction values are needed as well.